bucket/geogebra5.json

```diff
@@ -0,0 +1,38 @@
+{
+    "version": "5.0.675.0",
+    "description": "A dynamic mathematics software for education that brings together geometry, algebra, spreadsheets, graphing, statistics and calculus.",
+    "homepage": "https://www.geogebra.org/",
+    "license": {
+        "identifier": "freeware for non-commercial use",
+        "url": "https://www.geogebra.org/license"
+    },
+    "url": "https://download.geogebra.org/installers/5.0/GeoGebra-Windows-Portable-5-0-675-0.zip",
+    "hash": "0d6b8d21ee44e68ddd00337fbf7672af4318ad9fd3f1b88cd5230fd5f79d1163",
+    "installer": {
```

Comment:
Remove `installer`, because it can be replaced by `extract_dir`.

Reply:
See conversation below, it's not possible sadly.